pkg/cvo/metrics.go

```diff
@@ -399,13 +399,15 @@ func (m *operatorMetrics) Collect(ch chan<- prometheus.Metric) {
 		ch <- g
 	}
 
+	clusterId := getClusterId(m)
```

Comment:
There is no need to re-fetch clusterID every time we collect metrics. On line 254 we already check if `cv` can be fetched, so cluster ID is never `missing` (its being generated before cluster bootstrap)

Reply:
Updated @vrutkovs can you please also help me with the unit test, why does the added cv creates 3 more iterations of cluster_version instead of 1